gstreamer-pbutils/src/auto/discoverer_stream_info.rs

```diff
@@ -0,0 +1,91 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir @ 47eb915)
+// from gir-files (https://github.com/gtk-rs/gir-files @ ???)
+// DO NOT EDIT
+
+use ffi;
+use glib::object::IsA;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use gst;
+use std::mem;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct DiscovererStreamInfo(Object<ffi::GstDiscovererStreamInfo>);
+
+    match fn {
+        get_type => || ffi::gst_discoverer_stream_info_get_type(),
+    }
+}
+
+unsafe impl Send for DiscovererStreamInfo {}
+unsafe impl Sync for DiscovererStreamInfo {}
+
+pub trait DiscovererStreamInfoExt {
+    fn get_caps(&self) -> Option<gst::Caps>;
+
+    fn get_misc(&self) -> Option<gst::Structure>;
+
+    fn get_next(&self) -> Option<DiscovererStreamInfo>;
+
+    fn get_previous(&self) -> Option<DiscovererStreamInfo>;
```

Comment:
This looks like some kind of iterator impl around this would be nice. But that's for later :)